README.md

```diff
@@ -806,6 +948,467 @@ VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness vi
 UPLOAD - User's file is uploading
 ```
 
+## Premium Enterprise Features
```

Comment:
Out of scope: this readme is getting long, I wonder if it still makes sense to keep this info here or if we should create a separate .md file

Reply:
Agreed. I think we're eventually going to have all this on the Developer Hub & it should be broken up into smaller pages there, but in the meantime I can look into how we can break up the documentation here into separate Markdown files.